### PR TITLE
Vanity address generator

### LINKED ISSUE #87: [New tool] Vanity address generator
Similiar to: https://vanity-eth.tk/

It should be possible to generate a vanity address for development purposes. 

1. User provides a desired prefix (or it's empty)
2. Tools finds a private key/address pair where address begins with a desired prefix.

### PR DESCRIPTION
**Pull request by 7sne**
Closes #87

### CODE REVIEW

File: src/components/CopyableConversionInput.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ function InputCopy({ name, useState, ...props }: InputCopyProps): ReactElement {
                 .value;
             else throw new Error('Copyable ConversionInput must have a name');
             if (inputValue) {
-              await navigator.clipboard.writeText(inputValue);
+              void window.navigator.clipboard.writeText(inputValue);
```
Comment (krzkaczor) [inline src/components/CopyableConversionInput.tsx:60]: why this change?

**Review by krzkaczor — APPROVED** [review]

File: src/components/CopyableConversionInput.tsx (hunk 1 above)
Comment (7sne) [reply src/components/CopyableConversionInput.tsx:60]: This operation is async; await is meaningless in this context, we want to just call something without the need to get the result, thus void. Window for the code transparency.
